src/interop/js-interop/package-web.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Migrate to package:web
+description: How to migrate web interop code from dart:html to package:web.
+---
+
+[`package:web`][] exposes access to browser binding APIs,
+enabling interop between Dart applications and the web.
+Use `package:web` to program the browser and
+manipulate objects and elements in the DOM.
+
+```dart
+import 'package:web/web.dart';
+
+void main() {
+  final div = document.querySelector('div')!;
+  div.text = 'Text set at ${DateTime.now()}';
+}
+```
+
+`package:web` is generated automatically from [Web IDL][@webref/idl] declarations,
```

Comment:
> I think we're shying away from "static interop" going forward to reduce confusion.

Interesting, like, in all contexts or just this one for some reason? Why is that? 
I think I've been saying it a lot so I'll have to check all my recent work!

Reply:
It's mostly messaging really. Part of it is that we're saying users should really have one form of interop going forwards, so prefacing it with static is just confusing because for them, it's just interop. The other part is `@staticInterop` could be considered "static interop" but we don't want people using that now that extension types are out.

It's fine for us to say colloquially/internally to differentiate it from older interop.